run_with_hanskenpy.py

```diff
@@ -1,6 +1,6 @@
 from hansken_extraction_plugin.runtime.extraction_plugin_runner import run_with_hanskenpy
 
-from plugin.extraction_plugin import Plugin
+from plugin.plugin import Plugin
 
 if __name__ == '__main__':
```

Comment:
I like to simply put the `run_with_hanskenpy` call in the plugin itself within the `if __name__ == '__main__'` check. The `serve_plugin` doesn't seem to mind.

Reply:
https://github.com/NetherlandsForensicInstitute/audio-segmentation/blob/master/audio_segmentation.py#L197-L198

Reply:
will do!